fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/BaseWhereAdapter.java

```diff
@@ -0,0 +1,1137 @@
+/*
+ * (C) Copyright IBM Corp. 2019,2021
```

Comment:
```suggestion
 * (C) Copyright IBM Corp. 2021
```
seems like a new file (and typically separate year by comma-space)

Reply:
ok